template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/training/notebooks/TrainWithFeatureStore.py.tmpl

```diff
@@ -233,6 +264,30 @@ training_set = fs.create_training_set(
     exclude_columns=exclude_columns,
 )
 
+{{else}}
+from databricks.feature_engineering import FeatureEngineeringClient
+
+# End any existing runs (in the case this notebook is being run for a second time)
+mlflow.end_run()
+
+# Start an mlflow run, which is needed for the feature store to log the model
+mlflow.start_run()
+
+# Since the rounded timestamp columns would likely cause the model to overfit the data
+# unless additional feature engineering was performed, exclude them to avoid training on them.
+exclude_columns = ["rounded_pickup_datetime", "rounded_dropoff_datetime"]
+
+fe = FeatureEngineeringClient()
+
+# Create the training set that includes the raw input data merged with corresponding features from both feature tables
+training_set = fe.create_training_set(
+    df=taxi_data, # specify the df 
+    feature_lookups=pickup_feature_lookups + dropoff_feature_lookups, # both features need to be available; defined in GenerateAndWriteFeatures &/or feature-engineering-workflow-resource.yml
+    label="fare_amount",
+    exclude_columns=exclude_columns,
+)
+{{end -}}
```

Comment:
So the `-` at the end means to remove space after this line. You should try generating this notebook in your local Stacks repo by using `databricks bundle init` and add screenshots of both UC and non-UC to make sure they look as expected

Reply:
Will try generating -- I didn't realize I could! Thanks